python/lsst/pipe/tasks/calibrate.py

```diff
@@ -226,9 +255,6 @@ def __init__(self, icSourceSchema=None, **kwargs):
         """!Construct a CalibrateTask
 
         @param[in] icSourceSchema  schema for icSource catalog, or None.
-            If measuring aperture correction and the task detectAndMeasure cannot determine
-            its own suitable candidates, then this argument must be specified.
```

Comment:
Not sure these lines can be removed without explanation.  Can this value still be `None`?  Under what conditions may it be `None`?